packages/dcos-integration-test/extra/test_3dt.py

```diff
@@ -22,29 +20,18 @@ def make_3dt_request(cluster):
     returns a function that performs authenticated connections to 3DT on the appropriate port
     and appends ?cache=0 to all endpoints
     """
-    def make_request(ip, endpoint, is_agent):
+    def make_request(ip, endpoint):
         assert endpoint.startswith('/'), 'endpoint {} must start with /'.format(endpoint)
         endpoint = BASE_ENDPOINT_3DT + endpoint + '?cache=0'
-        scheme = urllib.parse.urlparse(cluster.dcos_uri).scheme
-        # TODO: SSL settings are not a part of dcos/dcos. Migrate settings to be in the same repo
-        port = {
-            'agent': {'http': 61001, 'https': 61002},
-            'master': {'http': 80, 'https': 443}}['agent' if is_agent else 'master'][scheme]
-        url = '{}://{}:{}/{}'.format(scheme, ip, port, endpoint.lstrip('/'))
-        logging.info('GET {}'.format(url))
-        request_headers = {}
-        if cluster.auth_enabled:
-            request_headers = cluster.web_auth_default_user.auth_header
-        response = requests.get(url, headers=request_headers)
+        response = cluster.get(path=endpoint, node=ip)
```

Comment:
nit: would have been nice in the commit message to have a comment that this is moving 3dt to go through agent adminrouter. Took me a while to figure that out.